colmena/task_server/parsl.py

```diff
@@ -402,6 +402,7 @@ def _submit(self, task: Result, topic: str) -> Optional[Future]:
             method = task.method
 
         # Submit the application
+        task.mark_task_submitted()
```

Comment:
We should emphasize that this the time we start the submission process. If we wanted the time after it's done submitting, we could store that information in the task server and then inject it into the result object before sending it back to the client.

Reply:
Makes sense. Would changing the name from `task_submitted` -> `start_task_submission` and updating the field description be sufficient?

Reply:
Yea, for the scope of this PR. Let's re-consider which ones we measure and how we describe them at some point